tokenRegistry/package.json

```diff
@@ -14,7 +14,7 @@
   },
   "main": "./dist/cjs/index.js",
   "module": "./dist/esm/index.js",
-  "types": "./dist/esm/index.d.ts",
+  "types": "./dist/cjs/index.d.ts",
```

Comment:
serious question... why not drop commonjs support and only use esm? Node has supported ESM for like 3 years now.

Reply:
fine w/ me but not this release plzz

Reply:
fair enough lmao